tests/functional/data/searches.py

```diff
@@ -0,0 +1,108 @@
+from dataclasses import dataclass
+from typing import List, Union
+
+
+@dataclass
+class Field:
+    path: str
+    expected_value: Union[str, int, float]
+
+
+@dataclass
+class Search:
+    query: List[tuple]
+    expected_response_fields: List[Field]
+
+
+DEFAULT = Search(query=[("family", "Capon"),
+                        ("gender", "male"),
+                        ("birthdate", "eq1953-05-29")],
+                 expected_response_fields=[Field("resourceType", "Bundle"),
+                                           Field("type", "searchset"),
+                                           Field("entry[0].resource.id", "9693632117")])
+
+SENSITIVE = Search(query=[("family", "Godsoe"),
+                          ("gender", "male"),
+                          ("birthdate", "eq1936-02-24")],
+                   expected_response_fields=[Field("entry[0].resource.id", "9693632125")])
+
+UNKNOWN_GENDER = Search(query=[("family", "Massam"), ("birthdate", "eq1920-08-11")],
+                        expected_response_fields=[Field("entry[0].resource.id", "9693632966")])
+
+DOB_RANGE = Search(query=[("family", "Massam"),
+                          ("birthdate", "le1920-08-11")],
+                   expected_response_fields=[Field("entry[0].resource.id", "9693632966")])
+
+VAGUE = Search(query=[("family", "YOUDS"),
```

Comment:
probably me being picky, but would MULTIPLE_MATCHES be a better descriptor than 'VAGUE'?

Reply:
I like the name, as depending on the access mode and search parameters, different results may come back. You could get multiple matches or you could get too many matches.